lib/_stream_readable.js

```diff
@@ -132,8 +132,10 @@ function ReadableState(options, stream, isDuplex) {
   // Everything else in the universe uses 'utf8', though.
   this.defaultEncoding = options.defaultEncoding || 'utf8';
 
-  // The number of writers that are awaiting a drain event in .pipe()s
-  this.awaitDrain = 0;
```

Comment:
Streams are heavily relied upon and we should run CITGM and check https://gzemnid.nodejs.org for the usage of `awaitDrain`.

Reply:
I think adding a legacy getter that returns the value of `awaitDrain` computed from the new properties wouldn’t be a bad idea…

Reply:
I just looked at `gzemnid` and all hits seem to originate from bundling `readable-streams`.

We could add a legacy getter but it seems like this is indeed not used.